src/plugins/syscalls/win.cpp

```diff
@@ -204,6 +204,7 @@ static event_response_t syscall_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
         }
     }
 
+    vmi.unlock();
```

Comment:
This is very fragile thing. It is better to avoid manual control of locks.

Reply:
@disaykin, @tklengyel I'm working on next branch where re-factor the function. I would prefer to make it there to simplify re-base.